app/panel/components/Panel.jsx

```diff
@@ -56,6 +60,15 @@ class Panel extends React.Component {
 				this.props.actions.updatePanelData(body);
 			}
 		});
+
+		chrome.runtime.onMessage.addListener((request) => {
+			if (request.name === 'showInsightsModal') {
+				this.setState({
+					...this.state,
+					insightsPromoModalShown: true
+				});
+			}
+		});
```

Comment:
We can actually delete this since all of the rendering logic is now in _renderInsightsPromoModal().  It must have snuck in when resolving merge conflicts.

Reply:
Cool, go ahead and delete